app/src/main/java/com/example/harvestfresh/SignupActivity.java

```diff
@@ -0,0 +1,75 @@
+package com.example.harvestfresh;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.parse.ParseException;
+import com.parse.ParseUser;
+import com.parse.SignUpCallback;
+
+public class SignupActivity extends AppCompatActivity {
+
+    private Button btnFinalSignup;
+    private EditText etSignupUsername;
+    private EditText etSignupPassword;
+    private EditText etConfirmPassword;
+    private static final String TAG = "SIGNUPACTIVITY";
```

Comment:
Nit: Make this tag easier to read "SignupActivity", or this.getClass().getSimpleName()

Reply:
Done!